website/contributors.ts

```diff
@@ -0,0 +1,116 @@
+import { fetch } from 'undici';
```

Comment:
![badge](https://img.shields.io/badge/review-memo-blue.svg)
Copied from [Qwik doc](https://github.com/builderio/qwik/blob/1d71aec752b5122512e13044d3fe47092d6e92a4/packages/docs/contributors.ts#L1-L95), and modified a littile bit (e.g. changing directory path and dealing with github personal token)

Reply:
To add contributors to index.mdx, we can simply execute `pnpm contributors` 😄 
https://github.com/fabian-hiller/valibot/pull/252/files#diff-fae242fbf77a8a9d52625664bc36ea12316586f8a716b41137d897a2b7e3df76R19-R20